pkg/config/config.go

```diff
@@ -147,6 +147,9 @@ const (
 	StatsEnabledKey       = "stats.enabled"
 	StatsAddressKey       = "stats.address"
 	StatsFlushIntervalKey = "stats.flush_interval"
+
+	SecurityAuditCheckIntervalKey     = "security.audit_check_interval"
+	DefaultSecurityAuditCheckInterval = 12 * time.Hour
```

Comment:
Maybe the configuration should be in a different resolution (something more human-readable such as minutes, hours ) just to prevent small values